be/src/vec/exec/file_scan_node.cpp

```diff
@@ -90,18 +115,102 @@ Status FileScanNode::open(RuntimeState* state) {
     RETURN_IF_ERROR(ExecNode::open(state));
     RETURN_IF_CANCELLED(state);
 
+    RETURN_IF_ERROR(_acquire_and_build_runtime_filter(state));
+
     RETURN_IF_ERROR(start_scanners());
 
     return Status::OK();
 }
 
+Status FileScanNode::_acquire_and_build_runtime_filter(RuntimeState* state) {
+    // acquire runtime filter
+    _runtime_filter_ctxs.resize(_runtime_filter_descs.size());
+    for (size_t i = 0; i < _runtime_filter_descs.size(); ++i) {
+        auto& filter_desc = _runtime_filter_descs[i];
+        IRuntimeFilter* runtime_filter = nullptr;
+        state->runtime_filter_mgr()->get_consume_filter(filter_desc.filter_id, &runtime_filter);
+        DCHECK(runtime_filter != nullptr);
```

Comment:
Will DCHECK(false) throw an exception? I didn't find its definition.

Reply:
No, this only take effect when compiling BE with DEBUG mode